Reqnroll/Infrastructure/MatchArgumentCalculator.cs

```diff
@@ -4,6 +4,12 @@
 
 namespace Reqnroll.Infrastructure
 {
+    // The ArgumentInfo structure holds information about arguments extracted from a Step match. The StartOffset indicates where in the step text the value of the argument begins in that string.
+    public class ArgumentInfo
```

Comment:
I would rename this to `MatchArgument` and make it constructor-initialized: `public MatchArgument(object value, int? startOffset = null)`. I would change the `StartOffset` to `int?` so that is can be used for table/docstring arguments as well. See next comment.

Reply:
Completed. See [03c175b](https://github.com/reqnroll/Reqnroll/pull/233/commits/03c175b3fa7501b8a2c72d7b2b2dc7c5e1adffcc)